<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_SystemSelection</class>
 <widget class="QDialog" name="Dialog_SystemSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="dialogTitle">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CHOOSE SYSTEM</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="separatorLine">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="selectorTitle">
       <property name="text">
        <string>System</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="selectorBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="detailTextBox"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="acceptButton">
       <property name="text">
        <string>ACCEPT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
